Anoosha_09

Anoosha-Sehar opened the pull request:
Terms from Genome Tracker.

Anoosha-Sehar committed:
Updated 'hot dog' as a parent class of 'frankfurter'

Anoosha-Sehar committed:
Updated 'chili dog'

Changed the parent class of 'chili dog' from 'frankfurter' to 'hot dog'

Anoosha-Sehar committed:
Added a new term 'corn dog'

Anoosha-Sehar committed:
Updated 'turkey frankfurter'

Added a definition, another parent class 'frankfurter' and an exact synonym 'turkey frank;.

Anoosha-Sehar committed:
Updated 'chicken frankfurter'

Added a definition, new parent term 'frankfurter' and an exact synonym 'chicken frank'

Anoosha-Sehar committed:
Added new term 'chia seed powder'.

Anoosha-Sehar committed:
Added new term 'random sample meat cut'

Anoosha-Sehar committed:
Added new term 'feather meal'

ddooley commented on src/ontology/foodon-edit.owl:
```diff
@@ -41964,6 +42025,18 @@ Dioscorea japonica is used for food.</obo:IAO_0000115>
+
+    <rdf:Description rdf:about="http://purl.obolibrary.org/obo/FOODON_03310576">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03302012"/>
+        <obo:IAO_0000115 xml:lang="en">A cooked and smoked sausage of chicken served in a bread roll.</obo:IAO_0000115>
```
The term should have a full label "chicken frankfurter" and an exact synonym "chicken frank".  As well the definition only has to reference parent class, so "A frankfurter made entirely of chicken".

Anoosha-Sehar replied:
This term is actually already present in FoodOn with a full label 'chicken frankfurter'. So it's more like an update, and that's why only added 'chicken frank' as an exact synonym. Agreed with the changes in definition, I'll do that. Thanks alot!

ddooley replied:
you are so right about synonym!  Sorry, my bad!

ddooley commented on src/ontology/foodon-edit.owl:
```diff
@@ -42474,6 +42547,18 @@ Dioscorea japonica is used for food.</obo:IAO_0000115>
+
+    <rdf:Description rdf:about="http://purl.obolibrary.org/obo/FOODON_03310934">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03302012"/>
+        <obo:IAO_0000115 xml:lang="en">A smoked sausage made of finely minced turkey meat and served in a bread roll.</obo:IAO_0000115>
```
a turkey frank isn't necessarily served in a bread roll, so take that part out of the definition; and same comments as for chicken frankfurter.

ddooley commented on src/ontology/foodon-edit.owl:
```diff
@@ -19743,6 +19796,14 @@ Dioscorea japonica is used for food.</obo:IAO_0000115>
 
+    <!-- http://purl.obolibrary.org/obo/FOODON_03302012 -->
+
+    <rdf:Description rdf:about="http://purl.obolibrary.org/obo/FOODON_03302012">
```
This isn't correct.  A frankfurter is just the sausage itself.  A frankfurter in a bun can't be a subclass of frankfurter.

ddooley commented on src/ontology/foodon-edit.owl:
```diff
@@ -18405,6 +18405,59 @@ Dioscorea japonica is used for food.</obo:IAO_0000115>
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003924 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003924">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00001009"/>
```
corndog can't be a child of hotdog.  It can go under "multicomponent meat product" for now though.

Anoosha-Sehar replied:
Right, but as per many sources, corndog is usually categorized as a hotdog. https://en.wikipedia.org/wiki/Corn_dog. Considering that, shouldn't 'corndog' be a child of both classes, 'hot dog' and 'multicomponent meat product'

ddooley replied:
We have to distinguish a "hot dog" that one buys in a package from a store refrigerator, from a "hot dog in a bun" which vendors serve.  But a corndog is neither of these and can't be a child of them.  A corndog has a hotdog inside it.

ddooley replied:
or as wikipedia says, usually has a hotdog inside it, but sometimes another kind of sausage I guess.

Anoosha-Sehar replied:
Right, I think that makes sense. For now, I'll add that under 'multicomponent meat product'. Thanks

mateolan replied:
Who let the dogs out? (sorry.)

ddooley commented on src/ontology/foodon-edit.owl:
```diff
@@ -18405,6 +18405,59 @@ Dioscorea japonica is used for food.</obo:IAO_0000115>
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003926 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003926">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/STATO_0000503"/>
```
"random sample meat cut" should just be a parent of "meat cut" .  The stato term is a sampling strategy, not the material entity that is a piece of meat selected a certain way ;) .

Anoosha-Sehar committed:
Updated the definition for 'chicken frankurter' as discussed.

Anoosha-Sehar committed:
Updated definition for 'turkey frankfurter' as discussed.

Anoosha-Sehar committed:
Updated the definition and parent term for 'corn dog'

Anoosha-Sehar committed:
Updated 'random sample meat (cut)'

For now, I have added it as a subclass of 'meat (cut)' instead of parent class of 'meat cut', as  I think there could also be a possibility of having 'non random sample meat (cut)' under 'meat (cut). But let me know your suggestion and I can update accordingly.
